components/director/pkg/tenant/entity.go

```diff
@@ -28,6 +28,12 @@ const (
 	Account Type = "account"
 	// Subaccount tenants must have a parent of type Account.
 	Subaccount Type = "subaccount"
+	// Organization tenants can be parents of Folder tenants.
```

Comment:
// Organization tenants can be parents of Folder tenants. ->  // Organization tenants can be parents of Folder or ResourceGroup tenants.